### DISCUSSION TITLE
Controller refactoring

### DISCUSSION THREAD

**Discussion by timoschilling** [activeadmin/activeadmin#7870] [Ideas]
The controllers should move form the `lib` path to into the `app/controller` path, ref #2906.

**Comment by javierjulio** [comment]
@timoschilling was the PR still correct in that all files should be moved into `app/controller` path? If so we can submit a new PR with that change as we are now developing for v2.

**Comment by ndbroadbent** [comment]
Hi @timoschilling, is there any chance that this auto reloading issue might be easier to fix on the latest version of Rails, using Zeitwerk? I'm still finding it a bit annoying that I need to restart my app after every change.

**Comment by timoschilling** [comment]
🤷‍♂️ I don't use active admin anymore in active projects since ~5 Years. Feel free to submit a PR if you like to work on that topic.

**Comment by javierjulio** [comment]
I'll move this to discussions for now. If possible to still do I would accept a PR for it. Thank you.
